docs/modeling/connectors/README.md

```diff
@@ -144,16 +144,41 @@ Events are stored in the `events` section of the connector JSON, for example in
     },
 ```
 
-### Connector images
-Connectors are deployed as separate Docker images with an application. The communication between the connector and the runtime bundle uses the [message broker](../../architecture/application.md#rabbit-mq), by default Rabbit MQ.
+#### Configuration parameters
+Configuration parameters are the environment variables specific to each connector instance that are used at deployment time. They set values such as the SMTP host and port the [email connector](../connectors/ootb/email.md) will use. 
 
-The images for the out of the box connectors supplied with Activiti Enterprise are pulled from [Quay.io](https://quay.io). Custom connectors can be hosted on another Docker registry as long as a secret is created in the Activiti Enterprise infrastructure namespace. Alternatively custom images can be hosted on a publicly hosted container repository such as [Docker Hub](https://hub.docker.com/).
+**Note** Additional configuration parameters can be specified at deployment time and existing parameters set during the modeling experience can be overridden. 
 
-#### Connector variables
-When [deploying an application](../../administrator/deploy/README.md) that contains a connector there is the option to specify environment variables for each connector in JSON format. 
+The basic properties for configuration parameters are: 
 
-The following is an example of setting the connector variables for the [email connector](../connectors/ootb/email.md):
+| Property | Description | Example | 
+| -------- | ----------- | ------- | 
+| `Name` | The name of the parameter | SLACK_XOXB |
+| `Description` | A free text description of what the parameter is for | The Slack bot user token. | 
+| `Value` | An optional default value for the parameter. This can be overridden at deployment time. | xoxb- | 
+| `Required` | Set whether the parameter requires a value when being used | true | 
+
+Configuration parameters are stored in the `config` section of the connector JSON, for example in an excerpt of the [Slack connector](../connectors/ootb/slack.md): 
 
 ```json
-{"EMAIL_HOST":"https://mysmtp.com","EMAIL_PORT":"8050", "EMAIL_USERNAME":"email-user", "EMAIL_PASSWORD":"email-user-password"}
+    "config": [
+        {
+            "name": "SLACK_XOXB",
+            "description": "Slack bot user token",
+            "value": "",
+            "required": true
+            
+        },
+        {
+            "name": "SLACK_XOXP",
+            "description": "Slack admin user token",
+            "value": "",
+            "required": true
+        }
+    ],
 ```
+
+### Connector images
+Connectors are deployed as separate Docker images with an application. The communication between the connector and the runtime bundle uses the [message broker](../../architecture/application.md#rabbit-mq), by default Rabbit MQ.
```

Comment:
Should the link be attached to the `Rabbit MQ`text instead of `message broker` as it points to the rabbit-mq page?

Reply:
This is a good spot but I think I will leave the link on message broker and potentially update the architecture page itself.